k8_cortx_cloud/logs-cortx-cloud.sh

```diff
@@ -23,6 +23,13 @@ date=$(date +%F_%H-%M)
 solution_yaml=${1:-"solution.yaml"}
 nodename=""
 pods_found=0
+size_limit="500MB"
+duration="P5D"
+binlogs="False"
+coredumps="False"
+stacktrace="False"
+all="True"
```

Comment:
default for 'all' should be 'False' only

Reply:
done